packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -1,68 +1,96 @@
 import clsx from 'clsx'
-import React, {ComponentPropsWithoutRef, PropsWithChildren} from 'react'
+import React, {PropsWithChildren} from 'react'
+import retroBackgroundSrc from '../../../../static/images/illustrations/retro-background.png'
+import standupBackgroundSrc from '../../../../static/images/illustrations/standup-background.png'
+import feedbackBackgroundSrc from '../../../../static/images/illustrations/feedback-background.png'
+import estimationBackgroundSrc from '../../../../static/images/illustrations/estimation-background.png'
+import strategyBackgroundSrc from '../../../../static/images/illustrations/strategy-background.png'
+import premortemBackgroundSrc from '../../../../static/images/illustrations/premortem-background.png'
+import postmortemBackgroundSrc from '../../../../static/images/illustrations/postmortem-background.png'
+import {upperFirst} from '../../utils/upperFirst'
+import {MeetingTypeEnum} from '../../__generated__/NewMeetingQuery.graphql'
+import {CategoryID, CATEGORY_TEXT_COLORS, MEETING_TYPE_TO_CATEGORY} from './Categories'
 
 export interface CardTheme {
   primary: string
   secondary: string
 }
 
-export const ActivityCardImage = (
-  props: PropsWithChildren<React.ImgHTMLAttributes<HTMLImageElement>>
-) => {
-  const {className, src} = props
+const backgroundImgMap = {
+  retrospective: retroBackgroundSrc,
+  standup: standupBackgroundSrc,
+  feedback: feedbackBackgroundSrc,
+  estimation: estimationBackgroundSrc,
+  strategy: strategyBackgroundSrc,
+  premortem: premortemBackgroundSrc,
+  postmortem: postmortemBackgroundSrc
+} as const
```

Comment:
+1 Looks like something that could go into `Categories.ts` to keep it in one place